jetstream/statistics.py

```diff
@@ -787,3 +791,42 @@ def transform(
                     )
                 )
         return StatisticResultCollection(results)
+
+
+@attr.s(auto_attribs=True)
+class PopulationRatio(Statistic):
+    numerator: str
+    denominator: str
+    confidence_interval: float = 0.95
+    drop_highest: float = 0.005
+    num_samples: int = 10000
+
+    def transform(
+        self,
+        df: DataFrame,
+        metric: str,
+        reference_branch: str,
+        experiment: Experiment,
+        analysis_basis: parser_metric.AnalysisBasis,
+        segment: str,
+    ) -> StatisticResultCollection:
+        critical_point = (1 - self.confidence_interval) / 2
+        summary_quantiles = (critical_point, 1 - critical_point)
+
+        ma_result = mozanalysis.frequentist_stats.bootstrap.compare_branches(
+            df,
+            col_label=[self.numerator, self.denominator],
+            ref_branch_label=reference_branch,
+            stat_fn=lambda data: {np.sum(data[:, 0]) / np.sum(data[:, 1])},
```

Comment:
Sorry this is a nit, but should we do instead:
```suggestion
            stat_fn=lambda data: {np.mean(data[:, 0]) / np.mean(data[:, 1])},
```
The output is identical, but maybe it will be more understandable to future-us / future data scientists? When we say "population ratio", it's a shortcut for "population ratio of means". @danielkberry